src/ontology/mondo-edit.obo

```diff
@@ -258278,6 +258302,7 @@ name: muscular dystrophy-dystroglycanopathy (congenital with intellectual disabi
 def: "An autosomal recessive inherited congenital muscular dystrophy caused by mutations in the POMT2 gene. It is characterized by mental retardation and mild structural brain abnormalities resulting from defective glycosylation of alpha-dystroglycan." [NCIT:C126690]
 synonym: "congenital muscular dystrophy-dystroglycanopathy with intellectual disability type B2" EXACT [NCIT:C126690]
 synonym: "congenital muscular dystrophy-dystroglycanopathy with mental retardation type B2" EXACT DEPRECATED [NCIT:C126690]
+synonym: "congenital muscular dystrophy-POMT2 related" EXACT CLINGEN_PREFERRED []
```

Comment:
could you please add a dbxref here:
https://clinicalgenome.org/affiliation/50061/